apsuite/orbcorr/si_calc_bump_matrix_at_bc_magnet.py

```diff
@@ -28,9 +30,21 @@ def calc_matrices(minsingval=0.2, mcidx=0, deltax=10e-6):
 
     idcs = np.array(
         [bpm1_sec_index, bpm2_sec_index,
-        160+bpm1_sec_index, 160+bpm2_sec_index])
+            160+bpm1_sec_index, 160+bpm2_sec_index])
     idcs += 8*mcidx
 
+    # remove closest BPMS
+    idlist = np.arange(0, 160, 1)
+    idcs_ignore = list()
+    for i in np.arange(n_bpms_out):
+        idcs_ignore.append(idlist[bpm1_sec_index + 8*mcidx - (i+1)])
+        idcs_ignore.append(idlist[bpm2_sec_index + 8*mcidx + (i+1)])
+    idcs_ignore = np.array(idcs_ignore)
+    idcs_ignore = np.tile(idcs_ignore, 2)
+    idcs_ignore[n_bpms_out*2:] += 160
+    if idcs_ignore.size != 0:
+        orbcorr.params.enbllistbpm[idcs_ignore] = False
+
```

Comment:
is this necessary? The code was thought to work even without removing BPMs. Did you notice something wrong?